x-pack/test/functional/services/ml/trained_models_table.ts

```diff
@@ -264,6 +527,25 @@ export function TrainedModelsTableProvider(
       );
     }
 
+    public async deployModelsContinue(expectedStep?: string) {
+      await testSubjects.exists('mlTrainedModelsInferencePipelineContinueButton');
+      const continueButton = await testSubjects.find(
+        'mlTrainedModelsInferencePipelineContinueButton'
+      );
+      await continueButton.isEnabled();
+      await continueButton.click();
+      if (expectedStep) {
+        await testSubjects.existOrFail(expectedStep);
+      }
+    }
+
+    public async assertDeployModelsCreateButton(expectedStep?: string) {
+      await testSubjects.exists('mlTrainedModelsInferencePipelineCreateButton');
```

Comment:
FYI it doesn't assert anything or wait for condition. If you need to wait for button to be visible, use timeout with `existOrFail`